### PR TITLE
Installed node-sass and compiled scss into css.

### PR DESCRIPTION
**Pull request by thinkbulecount2**
https://github.com/oaworks/oa-works/issues/1

Installed node-sass and converted current CSS into SCSS, then ran the command to compile back into CSS. Did some checking with the compiled CSS just to see if anything broke — none so far! Would love people's inputs.

### CODE REVIEW

**Comment by thinkbulecount2** [comment]
TODOs from the [issue](https://github.com/oaworks/oa-works/issues/1) as well as changes to the file structure as outlined here:

oa-works/
`--- src/ 
     |--- _data/ 
     |--- _includes/
     |--- about/ 
     |--- developers/ 
     |--- impact/ 
     |--- newsletter/ 
     |--- people/   
     |--- policies/  
     |--- products   
     `--- assets / 
          |--- css/
          |    |--- base/ 
          |    |    `--- _fonts.scss (..) _inverted.scss
          |    |--- layout / 
          |    |    `--- _sections.scss (..) _footer.scss
          |    |--- components/
          |    |    `--- _animations.scss (..) _timeline.scss
          |    |--- page/
          |    |    `--- _homepage.scss (..) _markdown.scss
          |    `--- main.scss
          |--- favicons
          |--- fonts
          |--- img 
          `--- js 

- [x]  Finish Gulp SCSS setup!
- [x]  Move all static asset directories and files to the structure shown above (JS, favicons, fonts, img)
- [x]  Fix up URL paths in templates (they’ll be probably a bit everywhere, might need to Regex that), ensure all of these static files still work, and test them visually
- [x]  Rename all .css files to be .scss partials prefixed with _ — i.e. homepage.css becomes homepage.scss
- [x]  Create main.scss in the new css/directory
- [x]  Basing yourself on the structure and order of the current main.html (found in oa-works/src/), import all individual scss in that main.scssfile using the https://github.com/import rule
- [ ]  Test that all CSS output still works. Ghost Inspector might come in handy with this — we can do this together on our next call
- [x]  Write documentation

File: .eleventy.js [hunk 1]
```diff
@@ -28,16 +28,16 @@ module.exports = function(eleventyConfig) {
...
   eleventyConfig.addPlugin(cacheBuster(cacheBusterOptions));
 
-  // Add CSS minifier
-  eleventyConfig.addFilter("cssmin", function(code) {
-    return new CleanCSS({}).minify(code).styles;
-  });
+  // Add CSS minifier -- replaced by a gulp task
+  // eleventyConfig.addFilter("cssmin", function(code) {
+  //   return new CleanCSS({}).minify(code).styles;
+  // });
```
Comment (sssoz) [inline .eleventy.js:34]: No need to comment out — OK to commit the change separately and write a message for this

File: .eleventy.js [hunk 2]
```diff
@@ -28,16 +28,16 @@ module.exports = function(eleventyConfig) {
...
 
   // Set directories to pass through to the dist folder
-  eleventyConfig.addPassthroughCopy('./src/img/');
-  eleventyConfig.addPassthroughCopy('./src/fonts/');
-  eleventyConfig.addPassthroughCopy('./src/js/');
-  eleventyConfig.addPassthroughCopy('./src/favicons/');
+  eleventyConfig.addPassthroughCopy('./src/assets/img/');
+  eleventyConfig.addPassthroughCopy('./src/assets/fonts/');
+  eleventyConfig.addPassthroughCopy('./src/assets/js/');
+  eleventyConfig.addPassthroughCopy('./src/assets/favicons/');
```
Comment (sssoz) [inline .eleventy.js:40]: Agreed that we should move everything to an `assets/` directory; could we do this with the `main.css` file as well? You will also need to update the path in `<head>`

**Review by sssoz — CHANGES_REQUESTED** [review]

**Comment by sssoz** [comment]
@thinkbulecount2 Another comment: now that static assets are in the `assets/` directory, we need to make sure all images use that path too. This needs to be done on the following pages (and check others to make sure): 

- Homepage
- Team
- Partners

Screenshot 2022-02-24 at 17 54 03

Screenshot 2022-02-24 at 17 53 31

Screenshot 2022-02-24 at 17 54 17

Screenshot 2022-02-24 at 17 54 27

**Comment by sssoz** [comment]
@thinkbulecount2 Once you’ve made the corrections and are ready to have this PR reviewed, please assign me to this PR!

**Comment by thinkbulecount2** [comment]
Will do @sssoz!

File: .eleventy.js (hunk 1 above)
Comment (sssoz) [reply .eleventy.js]: @thinkbulecount2 please resolve review comments once they are done!

**Comment by thinkbulecount2** [comment]
> 

I think I've covered everything, @sssoz! Let me know if I've missed anything!

**Comment by sssoz** [comment]
@thinkbulecount2 I’m still getting errors when running locally; the images aren’t loading: 

```
: https://oa.works/assets/img/stories/solar-panels.jpg
LazyImages - Input buffer has corrupt header: glib: XML parse error: Error domain 1 code 77 on line 554 column 34 of data: Premature end of data in tag html line 1
```

Could you check your setup locally as well and make sure the images load? I’ve deleted the `dist/` directory to ensure everything was re-compiled from scratch of course.

**Comment by sssoz** [comment]
Just realised the site.base_url variable is wrong! Let me adjust my local settings first. sorry!

**Comment by sssoz** [comment]
Just merged to develop to see if it works on dev.oa.works but it seems that stylesheets aren’t loading at all now: https://dev.oa.works/

**Comment by thinkbulecount2** [comment]
Just pushed an update, @sssoz, I think I've managed to fix all the snags.

**Comment by thinkbulecount2** [comment]
Don't know if I should create a new PR for the update — let me know which way works best, @sssoz!

**Comment by sssoz** [comment]
@thinkbulecount2 when you push to the remote branch, it should show up here. Have you tried to?

**Comment by thinkbulecount2** [comment]
@sssoz, I did, but if you see here it's asking me to create a new pull request instead of updating this one >>
Screenshot 2022-03-02 at 12 31 59

**Comment by sssoz** [comment]
Then in that case, go ahead and create one if that's what it's asking to do :)

**Pull request merged**